### PR TITLE
/etc/resolv.conf added to linux artifacts

### PR DESCRIPTION
**Pull request by chlete**
Could be useful to check if a DNS has been changed to one controlled by an attacker.

[Review on Reviewable](https://reviewable.io/reviews/forensicartifacts/artifacts/156)

### CODE REVIEW

File: definitions/linux.yaml [hunk 1]
```diff
@@ -450,3 +450,12 @@ urls:
   - 'https://www.kernel.org/doc/Documentation/filesystems/sysfs-pci.txt'
   - 'https://wiki.debian.org/HowToIdentifyADevice/PCI'
 supported_os: [Linux]
+---
+name: ResolvConfFile
```
Comment (destijl) [inline definitions/linux.yaml]: Lets stick DNS in the name to make it easier to discover:

DNSResolvConfFile
doc: DNS resolver configuration file

Also move it up so it's in alphabetical order with the other artifacts.

**Comment by destijl** [comment]
Thanks, just some minor changes.

**Comment by chlete** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[definitions/linux.yaml, line 454 \[r1\]](https://reviewable.io:443/reviews/forensicartifacts/artifacts/156#-K8y4uSXPjsC_0izCtS6-r1-454:-K9XbWrtT2PdpwadUYmK:-499146009) ([raw file](https://github.com/forensicartifacts/artifacts/blob/d448add408e76ca18c85b1745c9afe84ba945fba/definitions/linux.yaml#L454)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/forensicartifacts/artifacts/156)*

**Comment by destijl** [comment]
Thanks!

**Pull request merged**